yarn.lock

```diff
@@ -26631,21 +26631,11 @@ typescript-fsa@^2.0.0, typescript-fsa@^2.5.0:
   resolved "https://registry.yarnpkg.com/typescript-fsa/-/typescript-fsa-2.5.0.tgz#1baec01b5e8f5f34c322679d1327016e9e294faf"
   integrity sha1-G67AG16PXzTDImedEycBbp4pT68=
 
-typescript@^3.3.3333, typescript@~3.3.3333:
```

Comment:
I've been seeing this in a number of my branches after bootstrapping Kibana. I'll follow up with the team and figure out what's going on, but I am fairly certain we don't need these changes in this PR.